xc/device_common.ml

```diff
@@ -321,6 +321,10 @@ let qemu_restore_path : (_, _, _) format = "/var/lib/xen/qemu-resume.%d"
 let demu_save_path : (_, _, _) format = "/var/lib/xen/demu-save.%d"
 let demu_restore_path : (_, _, _) format = "/var/lib/xen/demu-resume.%d"
 
+let var_run_xen_path = "/var/run/xen"
+let qmp_libxl_path : (_, _, _) format = "/var/run/xen/qmp-libxl-%d"
+let qmp_event_path : (_, _, _) format = "/var/run/xen/qmp-event-%d"
```

Comment:
to avoid constant duplication here, shouldn't we write
```
+let qmp_libxl_path : (_, _, _) format = "%s/qmp-libxl-%d" var_run_xen_path
+let qmp_event_path : (_, _, _) format = "%s/qmp-event-%d" var_run_xen_path
```
instead?

Reply:
I like that.

Reply:
Updated, thanks.